src/Adapter/Order/CommandHandler/UpdateProductInOrderHandler.php

```diff
@@ -56,9 +58,29 @@ final class UpdateProductInOrderHandler extends AbstractOrderHandler implements
      */
     private $orderProductQuantityUpdater;
 
-    public function __construct(OrderProductQuantityUpdater $orderProductQuantityUpdater)
-    {
+    /**
+     * @var OrderAmountUpdater
+     */
+    private $orderAmountUpdater;
+
+    /**
+     * @var int
+     */
+    private $computingPrecision;
+
+    /**
+     * UpdateProductInOrderHandler constructor.
+     *
+     * @param OrderProductQuantityUpdater $orderProductQuantityUpdater
+     * @param OrderAmountUpdater $orderAmountUpdater
+     */
+    public function __construct(
+        OrderProductQuantityUpdater $orderProductQuantityUpdater,
+        OrderAmountUpdater $orderAmountUpdater
+    ) {
+        $this->computingPrecision = Context::getContext()->getComputingPrecision();
```

Comment:
Computing precision doesn't depend on the context, it must be computed based on the Order's currency (besides using `Context` class is ugly 😁)